core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/stack/detach/StackUpdateService.java

```diff
@@ -47,7 +47,6 @@ public void updateNameAndCrn(Stack stack, String newName, String newCrn) {
                 stack.setResourceCrn(newCrn);
                 Stack savedStack = measure(() -> stackService.save(stack),
                         LOGGER, "Stack save took {} ms for stack {}", stack.getName());
-                // While detaching the stack, name of the cluster is renamed to avoid conflicts.
```

Comment:
Not sure why I removed this I will add it back in later.

Reply:
Please revert this change.

Reply:
Done.